src/targets/typescript.ts

```diff
@@ -78,9 +78,10 @@ export default async function build({
       );
     }
 
-    let tsc =
-      path.join(root, 'node_modules', '.bin', 'tsc') +
-      (platform() === 'win32' ? '.cmd' : '');
+    const tscPath = options?.tsc
+      ? options.tsc
+      : path.join(root, 'node_modules', '.bin', 'tsc');
+    let tsc = tscPath + (platform() === 'win32' ? '.cmd' : '');
```

Comment:
I think if the user specified the path to `tsc`, we probably shouldn't modify it in any way to make it flexible (e.g. append `.cmd`). They can conditionally specify this path according to platform using `bob.config.js`.

Reply:
appending `.cmd` is now only done when no tsc option is provided